core/block_dragger.js

```diff
@@ -282,7 +282,7 @@ Blockly.BlockDragger.prototype.endBlockDrag = function(e, currentDragDeltaXY) {
           // Check for call blocks with no associated define block.
           if (!Blockly.Procedures.getDefineBlock(procCode, ws)) {
             // TODO:(#1151)
-            alert('To delete a block definition, first remove all uses of the block');
+            alert(Blockly.Msg.PROCEDURE_USED);
```

Comment:
This will only get the pop-up if you drag the definition off the workspace. There is also a version in the context menu procedure: https://github.com/LLK/scratch-blocks/blob/a355ca0b6353cbcfa906472d01a21e9940a390c9/blocks_vertical/vertical_extensions.js#L178-L180

Also, I'm not quite sure whether these are going to get dynamically updated if the language changes.